Formula/g/go.rb

```diff
@@ -63,21 +63,17 @@ class Go < Formula
   end
 
   def install
+    libexec.install Dir["*"]
     (buildpath/"gobootstrap").install resource("gobootstrap")
     ENV["GOROOT_BOOTSTRAP"] = buildpath/"gobootstrap"
 
-    cd "src" do
-      ENV["GOROOT_FINAL"] = libexec
+    cd libexec/"src" do
       # Set portable defaults for CC/CXX to be used by cgo
       with_env(CC: "cc", CXX: "c++") { system "./make.bash" }
     end
 
-    rm_r("gobootstrap") # Bootstrap not required beyond compile.
```

Comment:
Why would we want to preserve the downloaded Go bootstrap after using it to compile the wanted version?

Reply:
It isn't preserved. It is left behind in `/tmp` and gets wiped by brew automatically.